examples/keyword-extraction-api.ipynb

```diff
@@ -0,0 +1,615 @@
+       "    'object': 'chat.completion',\n",
+       "    'created': 1733779108,\n",
+       "    'model': 'klusterai/Meta-Llama-3.1-70B-Instruct-Turbo',\n",
+       "    'choices': [{'index': 0,\n",
+       "      'message': {'role': 'assistant',\n",
+       "       'content': '\"Earthquakes\", \"Prediction\", \"Geologists\", \"Subsoil Water\", \"Metals\"',\n",
+       "       'tool_calls': []},\n",
+       "      'logprobs': None,\n",
+       "      'finish_reason': 'stop',\n",
+       "      'stop_reason': None}],\n",
+       "    'usage': {'prompt_tokens': 107,\n",
+       "     'total_tokens': 130,\n",
+       "     'completion_tokens': 23},\n",
+       "    'prompt_logprobs': None}}},\n",
+       " {'id': '67575ea4d7432ed97b937dc9',\n",
+       "  'custom_id': 'keyword_extraction-4',\n",
+       "  'response': {'status_code': 200,\n",
+       "   'request_id': 'd1be58a5-3b9a-44a1-ab6e-591cee7a497e',\n",
+       "   'body': {'id': 'chat-fcda5718a1434d4b8d80f9dafe33a97e',\n",
+       "    'object': 'chat.completion',\n",
+       "    'created': 1733779108,\n",
+       "    'model': 'klusterai/Meta-Llama-3.1-70B-Instruct-Turbo',\n",
+       "    'choices': [{'index': 0,\n",
+       "      'message': {'role': 'assistant',\n",
+       "       'content': '\"Marine\", \"Expedition\", \"Species\", \"Atlantic Ocean\", \"Ecosystems\"',\n",
+       "       'tool_calls': []},\n",
+       "      'logprobs': None,\n",
+       "      'finish_reason': 'stop',\n",
+       "      'stop_reason': None}],\n",
+       "    'usage': {'prompt_tokens': 108,\n",
+       "     'total_tokens': 130,\n",
+       "     'completion_tokens': 22},\n",
+       "    'prompt_logprobs': None}}}]"
+      ]
+     },
+     "execution_count": 22,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "job = client.batches.retrieve(inference_job.id)\n",
+    "result_file_id = job.output_file_id\n",
+    "result = client.files.content(result_file_id).content\n",
+    "parse_json_objects(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "70e0e816-6558-4ff2-bab2-f85cff00bfc8",
+   "metadata": {
+    "executionInfo": {
+     "elapsed": 5,
+     "status": "ok",
+     "timestamp": 1732042430093,
+     "user": {
+      "displayName": "Joaquin Rodríguez",
+      "userId": "09993043682054067997"
+     },
+     "user_tz": 180
+    },
+    "id": "tu2R8dGYimKc"
+   },
+   "source": [
+    "## Conclusion"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d195dd1-1293-4407-b6ad-cab7e77b14c0",
+   "metadata": {},
+   "source": [
+    "Congratulations! You’ve successfully completed the keyword extraction task using the kluster.ai Adaptive Inference API. This demonstration highlights how you can effortlessly manage large datasets and extract meaningful insights from them. With the Adaptive Inference API, you can scale your workflows smoothly, making it an essential tool for processing large-scale datasets."
```

Comment:
let's continue to use Batch API until we get the go-ahead ->    "Congratulations! You’ve successfully completed the keyword extraction task using the kluster.ai Batch API. This demonstration shows how you effortlessly manage large datasets and extract meaningful insights from them. With the Batch API, you can scale your workflows smoothly, making it an essential tool for processing large-scale datasets."